dbf-reader/src/main/java/org/jamel/dbf/structure/DbfRow.java

```diff
@@ -0,0 +1,158 @@
+        Object value = get(fieldName);
+        return value != null && (boolean) value;
+    }
+
+    /**
+     * Retrieves the value of the designated field as int.
+     *
+     * @param fieldName the name of the field
+     * @return the field value, or 0 (if the dbf value is NULL)
+     * @throws DbfException if there's no field with name fieldName
+     */
+    public int getInt(String fieldName) throws DbfException {
+        return getNumber(fieldName).intValue();
+    }
+
+    /**
+     * Retrieves the value of the designated field as short.
+     *
+     * @param fieldName the name of the field
+     * @return the field value, or 0 (if the dbf value is NULL)
+     * @throws DbfException if there's no field with name fieldName
+     */
+    public short getShort(String fieldName) throws DbfException {
+        return getNumber(fieldName).shortValue();
+    }
+
+    /**
+     * Retrieves the value of the designated field as byte.
+     *
+     * @param fieldName the name of the field
+     * @return the field value, or 0 (if the dbf value is NULL)
+     * @throws DbfException if there's no field with name fieldName
+     */
+    public byte getByte(String fieldName) throws DbfException {
+        return getNumber(fieldName).byteValue();
+    }
+
+    /**
+     * Retrieves the value of the designated field as long.
+     *
+     * @param fieldName the name of the field
+     * @return the field value, or 0 (if the dbf value is NULL)
+     * @throws DbfException if there's no field with name fieldName
+     */
+    public long getLong(String fieldName) throws DbfException {
+        return getNumber(fieldName).longValue();
+    }
+
+    /**
+     * Retrieves the value of the designated field as float.
+     *
+     * @param fieldName the name of the field
+     * @return the field value, or 0 (if the dbf value is NULL)
+     * @throws DbfException if there's no field with name fieldName
+     */
+    public float getFloat(String fieldName) throws DbfException {
+        return getNumber(fieldName).floatValue();
+    }
+
+    /**
+     * Retrieves the value of the designated field as double.
+     *
+     * @param fieldName the name of the field
+     * @return the field value, or 0 (if the dbf value is NULL)
+     * @throws DbfException if there's no field with name fieldName
+     */
+    public double getDouble(String fieldName) throws DbfException {
+        return getNumber(fieldName).doubleValue();
+    }
+
+    /**
+     * Retrieves the value of the designated field as Object.
+     *
+     * @param fieldName the name of the field
+     * @return the field value, or null (if the dbf value is NULL)
+     * @throws DbfException if there's no field with name fieldName
+     */
+    public Object getObject(String fieldName) throws DbfException {
+        return get(fieldName);
+    }
+
+    private Number getNumber(String fieldName) {
+        Number value = (Number) get(fieldName);
+        return value == null ? 0 : value;
```

Comment:
Please make and use private constant `Long ZERO = 0L;` to eliminate useless boxing.